components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -98,11 +98,21 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) UIView *titleView;
 
+/**
+ The font applied to the title of navigation bar.
+ Font size is enforced to 20.
+ Default is MDCTypography's titleFont.
+ Note that Font attribute will of titleTextAttributes take precedence over titleFont property.
```

Comment:
There are two things we need to document here:

1. What the default, initial value is.
2. What happens when we set the property to nil.

Reply:
Done.